src/lib/temple/back/evm-validation-schemas/index.ts

```diff
@@ -48,6 +49,37 @@ export const ethPersonalSignPayloadValidationSchema = mixedSchema<
   return false;
 }).required();
 
+export const addEthAssetPayloadValidationSchema = objectSchema()
+  .shape({
+    type: stringSchema().required(),
+    options: objectSchema()
+      .shape({
+        address: evmAddressValidationSchema().required()
+      })
+      .required()
+  })
+  .required();
+
+export const addEthChainPayloadValidationSchema = tupleSchema([
+  objectSchema()
+    .shape({
+      chainId: stringSchema().required(),
+      chainName: stringSchema().required(),
+      nativeCurrency: objectSchema()
+        .shape({
+          name: stringSchema().required(),
+          symbol: stringSchema().required(),
+          decimals: numberSchema().integer().positive().required()
+        })
+        .required(),
+      rpcUrls: arraySchema().of(stringSchema().required()).required(),
```

Comment:
Use `src/lib/temple/back/evm-validation-schemas/utils.ts` to add common types there